install/pre-install-payload/payload.sh

```diff
@@ -13,7 +13,7 @@ official_containerd_version=${official_containerd_version:-"1.7.7"}
 vfio_gpu_containerd_repo=${vfio_gpu_containerd_repo:-"https://github.com/confidential-containers/containerd"}
 vfio_gpu_containerd_version=${vfio_gpu_containerd_version:-"1.7.0.0"}
 nydus_snapshotter_repo=${nydus_snapshotter_repo:-"https://github.com/containerd/nydus-snapshotter"}
-nydus_snapshotter_version=${nydus_snapshotter_version:-"v0.13.3-multiarch"}
+nydus_snapshotter_version=${nydus_snapshotter_version:-"v0.13.13"}
```

Comment:
is it necessary to keep updating the default when we have it specified in the `Makefile`? Having 2 places can be confusing...

Reply:
I agree with you, but didn't want to go into that on this PR. Do you thing an approach of us letting the `set -o nounset` fail if it's not passed in would make sense?

Reply:
@ldoktor - I've added a second commit, so please check that out and see if you are happy and we can get @fidencio to re-review too.